migrations/V117__Add_ TRB_Request_Attendee_Table.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE trb_request_attendees (
+    id UUID PRIMARY KEY NOT NULL,
+    trb_request_id uuid NOT NULL REFERENCES trb_request(id),
+    eua_user_id TEXT NOT NULL CHECK (eua_user_id ~ '^[A-Z0-9]{4}$'),
+    component TEXT NOT NULL,
+    role TEXT NOT NULL,
```

Comment:
I think we should probably make role (and other dropdowns, if possible) into enumerated values (from a SQL and GQL perspective).